### PR TITLE
Added ldap support

### PR DESCRIPTION
**Pull request by tobybatch**
Adds (the *very* popular) ldap support

### CODE REVIEW

**Comment by Schrolli91** [comment]
Please give me a little while to review this

File: Dockerfile [hunk 1]
```diff
@@ -39,37 +39,41 @@ RUN mkdir /opt/kimai && \
 #fpm alpine php extension base
 FROM php:7.3.10-fpm-alpine3.10 AS fpm-alpine-php-ext-base
 RUN apk add --no-cache \
-    # gd
```
Comment (Schrolli91) [inline Dockerfile:42]: have you sort the tools alphabetical or what happens here?

File: Dockerfile [hunk 2]
```diff
@@ -39,37 +39,41 @@ RUN mkdir /opt/kimai && \
...
-    g++
+    perl \
+    re2c \
+    # gd
+    freetype-dev \
+    libpng-dev \
+    # icu
+    icu-dev \
+    # ldap
+    icu-dev \
```
Comment (Schrolli91) [inline Dockerfile]: duplicate (see line 64)

File: Dockerfile [hunk 3]
```diff
@@ -39,37 +39,41 @@ RUN mkdir /opt/kimai && \
...
+    openldap-dev \
```
Comment (Schrolli91) [inline Dockerfile:66]: is this the only new dep?
after the reordering its not as easy to see

File: Dockerfile [hunk 4]
```diff
@@ -94,8 +98,9 @@ RUN docker-php-ext-install -j$(nproc) pdo_mysql
 FROM ${BASE}-php-ext-base AS php-ext-zip
 RUN docker-php-ext-install -j$(nproc) zip
 
-
-
+# php extension zip
```
Comment (Schrolli91) [inline Dockerfile]: update comment here pls

**Review by Schrolli91 — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (tobybatch) [reply Dockerfile:42]: I sorted the groups alphabetically.

File: Dockerfile (hunk 3 above)
Comment (tobybatch) [reply Dockerfile:66]: yep

File: Dockerfile (hunk 1 above)
Comment (tobybatch) [reply Dockerfile:42]: I guess they should really all be alphabetical

File: Dockerfile [hunk 5]
```diff
@@ -180,6 +184,9 @@ COPY --from=php-ext-gd /usr/local/lib/php/extensions/no-debug-non-zts-20180731/g
 # PHP extension intl
 COPY --from=php-ext-intl /usr/local/etc/php/conf.d/docker-php-ext-intl.ini /usr/local/etc/php/conf.d/docker-php-ext-intl.ini
 COPY --from=php-ext-intl /usr/local/lib/php/extensions/no-debug-non-zts-20180731/intl.so /usr/local/lib/php/extensions/no-debug-non-zts-20180731/intl.so
+# PHP extension ldap
+COPY --from=php-ext-ldap /usr/local/etc/php/conf.d/docker-php-ext-ldap.ini /usr/local/etc/php/conf.d/docker-php-ext-ldap.ini
```
Comment (Schrolli91) [inline Dockerfile]: We have to check how long the ldap extension build will took.
i have ordered the copy's from fastest to longest build time.
so the finished ones can be copied while the others still building (multitasking)

Comment (tobybatch) [reply Dockerfile]: @Schrolli91 I also added the time the builds took on my machine for later reference.

**Review by Schrolli91 — APPROVED** [review]
LGTM

**Pull request merged**